buildSrc/src/main/groovy/io.micronaut.internal.starter.base-module.gradle

```diff
@@ -15,6 +15,7 @@ ext {
 
 configurations.all {
     resolutionStrategy {
+        force "io.micronaut:micronaut-core-bom:${rootProject.micronautCoreVersion}"
```

Comment:
We shouldn't need this, but _something_ is constraining micronaut-core-bom to SNAPSHOT, so we end up with M2 and SNAPSHOT dependencies on the classpath

I'm looking into why this happens, but I'm not currently sure @melix does anything jump out at you?

Reply:
Well, M2 < SNAPSHOT, so Gradle will choose SNAPSHOT if you have it somewhere. "we end up with M2 and SNAPSHOT dependencies on the classpath" can you tell how you see this ?

Reply:
without this workaround ./gradlew :stater-core:dependencies

Reply:
This task wouldn't show duplicate entries on classpath. It would show that 2 different dependency graphs use different versions of core, which can be expected (especially with AOT which needs to run with a different classpath). That doesn't mean that we would end up with 2 different versions on classpath (which can, but shouldn't, be the composition of multiple graphs).